src/Http/Response/Format/Format.php

```diff
@@ -47,6 +54,18 @@ public function setResponse($response)
     }
 
     /**
+     * Set the formats' options.
+     *
+     * @param array $options
+     *
+     * @return void
+     */
+    public function setOptions(array $options)
+    {
+        $this->options = $options;
```

Comment:
Okey. For now it is not used as fluent, but (maybe) in the future it will. I'd added this.